### PR TITLE
fix(Dropdown|Modal|TextArea): update refs handlers

### PR DESCRIPTION
**Pull request by layershifter**
Addresses #607.

### CODE REVIEW

File: src/addons/TextArea/TextArea.js [hunk 1]
```diff
@@ -60,6 +60,8 @@ class TextArea extends Component {
     this.updateHeight(e.target)
   }
 
+  handleRef = c => (this.rootNode = c)
```
Comment (levithomason) [inline src/addons/TextArea/TextArea.js]: Nice, we should also standardize these ref variable names.  We currently use `*Node`, `_*` and `*Ref` in various places.

A ref can either be a node (DOM component) or a reference to a class instance (Composite component).  Due to this, I vote we go with `this.*Ref` for clarity and accuracy.

**Review by levithomason — CHANGES_REQUESTED** [review]

File: src/addons/TextArea/TextArea.js (hunk 1 above)
Comment (layershifter) [reply src/addons/TextArea/TextArea.js]: Yep, it makes sense 👍

Do I have to change the names to these?
```diff
-handleRef = c => (this.rootNode = c)
+handleRef = c => (this.textAreaRef = c)
```
```diff
-handleSearchRef = c => (this._search = c)
+handleSearchRef = c => (this.searchRef = c)
```

Comment (levithomason) [reply src/addons/TextArea/TextArea.js]: Looks right to my eyes 👍

**Review by levithomason — APPROVED** [review]

**Comment by levithomason** [comment]
I've updated the refs per our agreement.  Refs at the root I simply used `this.ref` and `this.handleRef` for the naming convention.  Mirrors how we handle `onClick` `this.handleClick` vs `onItemClick` `this.handleItemClick`.

Give it a look over and let me know if we're good to go here.

**Comment by layershifter** [comment]
I've updated `Checkbox` handler. LGTM :+1:

**Comment by levithomason** [comment]
Released in `semantic-ui-react@0.67`.

**Pull request merged**